app/src/main/java/com/github/sdp/ratemyepfl/viewmodel/RoomReviewsListViewModel.kt

```diff
@@ -27,7 +29,7 @@ class RoomReviewsListViewModel @Inject constructor(
 
     private fun updateReviewsList() {
         viewModelScope.launch {
-            reviewsLiveData.value = reviewsRepository.getByReviewableId(id)
+            reviewsLiveData.value = reviewsRepository.getByReviewableId(room!!.id)
```

Comment:
I wonder why `room` is non-nullable here and `course` is nullable `CourseReviewListViewModel`

Reply:
Oh, yes it's the same thing, I will unify and put both as nullable.